### ISSUE TITLE
Closing the expanding menu with keyboard only requires tabbing backwards

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwenty#354]
When you reach the last menu item of the open, expanded menu (in this case a social link), the next item when you tab is on the page. It is difficult the see where the focus is, because the menu remains open. 

Expected: The next item should be the close button.

Currently, to reach the close button we need to tab backwards through all the menu items.
